### ISSUE TITLE
`no-attribute-formation` Incorrectly Flags Atoms and `λ` Formations

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/lints#514]
In `number.eo` object in `eo-runtime` which is:
```
[as-bytes] > number
  ...
  [] > as-i64 ?
```

I see this:
```
[WARNING] org.eolang.number:31 The formation "as-i64" does not have any void attributes (no-attribute-formation)
[WARNING] org.eolang.number:31 The formation "λ" does not have any void attributes (no-attribute-formation)
```

I think this lint should not complain about atoms and `λ` formations (which is part of atom) at all

**Comment by maxonfjvipon** [comment]
@yegor256 @h1alexbel WDYT?

**Comment by h1alexbel** [comment]
@maxonfjvipon atom formation void attributes are the same attributes we pass to the normal formations, so the `no-attribute-formation` check is right, even though its experimental. About `λ`, agree, we should not complain. WDYT?

**Comment by 0crat** [comment]
@maxonfjvipon Hello! Thank you for reporting a new bug in our project. As per our team's policy, you've earned **8** points for this valuable contribution. Your keen eye helps improve our project's quality. If you notice anything else that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) up to par, please [don't hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your current balance stands at [+100](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep up the great work!

**Comment by h1alexbel** [comment]
@maxonfjvipon please try new version: [0.0.48](https://github.com/objectionary/lints/releases/tag/0.0.48). It should solve the issue

**Comment by h1alexbel** [comment]
@maxonfjvipon please try new version: [0.0.48](https://github.com/objectionary/lints/releases/tag/0.0.48). It should solve the issue
